Makefile

```diff
@@ -0,0 +1,14 @@
+container:
+	@docker-compose build test
+travis:
+	@go get -v -d -t
+test:
+	@docker-compose run --rm test
```

Comment:
Why make docker (and docker-compose) a requirement for tests?

Reply:
dunno, it's how it was setup /shrug i just made it keep working